src/components/Game/Game.tsx

```diff
@@ -21,10 +21,18 @@ const generateObstacle = (): ObstacleEntity => {
 };
 export const Game = ({ pokemonPlayer }: Props): JSX.Element => {
   const [playerCoord, setPlayerCoord] = useState(0);
+
   const [obstacles, setObstacles] = useState<ObstacleEntity[]>([]);
   const [isGameSessionStarted, setIsGameSessionStarted] = useState(false);
   const [score, setScore] = useState(0);
   const [paused, setPaused] = useState(false);
+
+  useEffect(() => {
+    if (playerCoord <= -60) {
+      document.dispatchEvent(setDownAI);
```

Comment:
It is a bad way of passing events between components in React app. You should use callbacks.